### PR TITLE
openutau: 0.1.158 -> 0.1.327

### LINKED ISSUE #253596: Update request: openutau 0.1.158 → 0.1.201
- Package name: openutau
- Latest released version:
- Current version on the unstable channel: 0.1.158
- Current version on the stable/release channel: -
- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**
@lilyinstarlight

I don't mind sending a PR to bump the version. I've tried building OpenUtau-0.1.201 on my device, and it works just like the previous instances.

The app, however, crashes, when I use any key, just like the current version, for which I've created stakira/OpenUtau#822.

-----

Note for maintainers: Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by lilyinstarlight**
## Description of changes

Bump version and remove now-unnecessary patches

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by majiru** [comment]
Result of `nixpkgs-review pr 274049` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>openutau</li>
  </ul>
</details>

**Review by majiru — APPROVED** [review]

File: pkgs/applications/audio/openutau/default.nix [hunk 1]
```diff
@@ -58,11 +44,15 @@ buildDotnetModule rec {
...
-  # needed until upstream bumps to dotnet 7
+  # net7.0 replacement needed until upstream bumps to dotnet 7
   postPatch = ''
     substituteInPlace OpenUtau/OpenUtau.csproj OpenUtau.Test/OpenUtau.Test.csproj --replace \
       '<TargetFramework>net6.0</TargetFramework>' \
       '<TargetFramework>net7.0</TargetFramework>'
+
+    substituteInPlace OpenUtau/Program.cs --replace \
+      '/usr/bin/fc-match' \
+      '${lib.getBin fontconfig}/bin/fc-match'
```
Comment (OPNA2608) [inline pkgs/applications/audio/openutau/default.nix:55]:
```suggestion
      '${lib.getExe' fontconfig "fc-match"}'
```

**Review by OPNA2608 — CHANGES_REQUESTED** [review]
Result of `nixpkgs-review pr 274049` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>openutau</li>
  </ul>
</details>

Don't have a voice bank to throw at it, but it launches fine & audio test works. Just one small suggestion, otherwise LGTM.

**Review by OPNA2608 — APPROVED** [review]
LGTM!

**Pull request merged**